docs/competition/livestream/chat-moderation.md

```diff
@@ -0,0 +1,51 @@
+# Chat Moderation
+
+We want our chat to be casual, fun and welcoming to all.
+
+For each livestreamed event, a 'Head Moderator' should be chosen. In the event of an edge case or an unclear next step, the Head Moderator will decide what action to take.
+
+If a moderator needs to enforce the rules against a user, a discussion between the moderators is encouraged to ensure a consistent moderation approach.
+
+## Dealing with a problem user
+
+Below is the escalation steps for dealing with a problem chat user
+
+### Engage
+
+First, try and engage with the user, talk about robots or other benign small-talk. In the ideal case, the user will start engaging with the stream and stop causing problems.
+
+### Warn
+
+If the problem behaviour continues, explicitly warn the user(s) to stop. Make it clear that a continuation of the behaviour will result in removal from the chat.
+
+### Timeout
+
+If the problem behaviour _still_ continues, timeout the problem user(s). As you've already warned them of this, you don't need to warn them or explain yourself again.
+
+### Ban
+
+If after the timeout there is still problem behaviour, ban the user. There is no need to warn or explain yourself for this.
+
+## When to bypass the standard escalation policies
+
+### Racism, Sexism, Homophobia, etc.
+
+Any comments of this kind should result in an instant ban. Delete the message and post a message in chat reminding everyone of the rules.
+
+### Flooding
+
+Timeout the user and remind everyone of the rules. The second offence should result in a ban.
+
+## Rules
+
+If the livestreaming platform does not enforce users accepting the rules before gaining chat access, these rules should be posted periodically in the chat.
+
+Mods and viewers must follow these rules:
+
+- No racism, sexism, homophobia etc.
+- No inappropriate comments; this is a family-friendly chat.
```

Comment:
Does this include swearing?

Reply:
This was intended to include swearing, hence the family-friendly comment. We also wanted it cover things like a user insulting another user without swears ('stupid', 'idiot' etc.). Open to explicitly mentioning swearing if others think its useful.